app/models/course_instance.rb

```diff
@@ -18,6 +18,10 @@ class CourseInstance < ActiveRecord::Base
     .unenrolled(user)
   }
 
+  def schedule
+    meetings.map(&:schedule).join('; ') || 'TBA'
```

Comment:
Afraid not. Calling `.join` is always going to convert it to a string, and strings are truthy.
`[].join('; ') == ''`
`'' || 'TBA' == ''`